lib/js/components/Pagination/Pagination.vue

```diff
@@ -333,8 +366,23 @@ export default {
 
 			return navigationItems;
 		},
+		navigationItemsForDropdown() {
+			if (this.lastPage <= FIRST_PAGE_NUMBER) {
```

Comment:
Shouldn't this be `<= MAX_NAVIGATION_ITEMS`? We won't be rendering dropdown if there is no ellipsis. See https://github.com/bethinkpl/design-system/blob/09ec502a7473b72ebdd8172041c9e8d211cfb4d9/lib/js/components/Pagination/Pagination.vue#L327-L334